src/main/java/hudson/remoting/Channel.java

```diff
@@ -291,7 +294,7 @@
     /**
      * Property bag that contains application-specific stuff.
      */
-    private final Hashtable<Object,Object> properties = new Hashtable<Object,Object>();
+    private final ConcurrentHashMap<Object,Object> properties = new ConcurrentHashMap<>();
```

Comment:
It could, will change though there is no much difference since the field is private